influxdb3/tests/server/auth.rs

```diff
@@ -143,7 +143,8 @@ async fn auth_grpc() {
 
         // Set the authorization header on the client:
         client
-            .add_header(header, &format!("Bearer {TOKEN}"))
```

Comment:
Almost all the other tests use `Bearer`, except for this one which I changed to use `Token`

Reply:
As an aside, this actually isn't testing the code you added, since this test is using the gRPC server, which handles token extraction in the core `authz` crate here: https://github.com/influxdata/influxdb3_core/blob/1eaa4ed5ea147bc24db98d9686e457c124dfd5b7/authz/src/lib.rs#L27-L44

If you want to test the new code you added, you would need to check using the HTTP server, e.g., in this test: https://github.com/influxdata/influxdb/blob/c4514bf401705ae86bbbb3551944cd2ee34152a2/influxdb3/tests/server/auth.rs#L8-L9

Reply:
Thanks @hiltontj - given both http 2/http 1 runs on same port, I wasn't sure if auth handling was shared. I'll update the test.